components/conversation/conversation.js

```diff
@@ -9,28 +9,29 @@ const api = require('./apiHelpers.js');
 
 const Conversation = ({ userId, friendInfo, chatId, handleBackButtonPress }) => {
   api.setConversationInfo(friendInfo.friendId, friendInfo.friendFirstName, friendInfo.friendAvatar);
+  let [currentChatId, setCurrentChatId] = useState(chatId);
   let [messages, setMessages] = useState([]);
   let [spotlightPic, setSpotlightPic] = useState('');
   let [picDisplay, setPicDisplay] = useState(false);
   let [cameraDisplay, setCameraDisplay] = useState(false);
   let [progressBarFill, setProgressBarFill] = useState(1);
 
   let screenShotListener = ScreenCapture.addScreenshotListener(() => {
-    api.noteScreenShot(chatId, userId);
+    api.noteScreenShot(currentChatId, userId);
   });
 
   // updates messages upon render
   useEffect(() => {
     async function updateMessages() {
-      const incomingMessages = await api.fetchMessages(chatId, userId);
+      const incomingMessages = await api.fetchMessages(currentChatId, userId);
```

Comment:
So does this trigger an update each time a new message comes in? If so that is rad